src/System.Runtime.Numerics/tests/ComplexTests.cs

```diff
@@ -353,14 +360,6 @@ public static IEnumerable<object[]> ASin_Advanced_TestData()
             yield return new object[] { -1234000000, 0, -1.5707963267948966, 21.62667394298955 }; // Real part is negative, imaginary part is 0
             yield return new object[] { 0, 1234000000, 0, 21.62667394298955 }; // Imaginary part is positive
 
-            // Boundary values
-            yield return new object[] { double.MaxValue, 0, double.NaN, double.NaN };
```

Comment:
I think it would be better to fix rather than remove these tests. That the previous implementation couldn't deal with the values well perhaps points to them being particularly good guards against a future change that reduced accuracy.